client/src/components/LeafletOsmMap/index.js

```diff
@@ -184,6 +192,8 @@ export default class LeafletOSMMap extends Component {
         //remove old icon
         droppedPin.remove()
 
+        //TODO: delete temporary marker from unSavedMarker
```

Comment:
isn't this currently handled in the `onMapClick()` function? I don't think think we need to implement this.